CHANGELOG.md

```diff
@@ -2,11 +2,13 @@
 
 ## 0.x - UNRELEASED
 
+
```

Comment:
Please keep whitespace changes to a minimum

Reply:
@gollth Sorry I overlooked that one. I will remove it.

Reply:
Fixed I will now apply your suggestions to the pull request.